src/shared/vars.js

```diff
@@ -1,4 +1,6 @@
-const VERSION = '__skate_0_14_0';
```

Comment:
don't worry, I thought this was just used for `window.skate.version`, but I see it has other uses now